### PR TITLE
better local dev experience

### PR DESCRIPTION
**Pull request by jzerbe**
- split out some functionality into `app.js`
- spun up test app with different app id for development
- use debug FB sdk on localhost
- added `site.yaml` manifest for drydrop prod deploy to pull from. pretty sure I got it right?

### CODE REVIEW

**Comment by jzerbe** [comment]
@jakect merge at your leisure

File: index.html [hunk 1]
```diff
@@ -2,99 +2,37 @@
...
 
         (function(d, s, id){
             var js, fjs = d.getElementsByTagName(s)[0];
             if (d.getElementById(id)) {return;}
             js = d.createElement(s); js.id = id;
-            // Used for local dev.
-            //js.src = "https://connect.facebook.net/en_US/sdk/debug.js";
-            js.src = "//connect.facebook.net/en_US/sdk.js";
+            if (isDev()) {
+                js.src = "//connect.facebook.net/en_US/sdk/debug.js";
```
Comment (jzerbe) [inline index.html]: :point_up: http://stackoverflow.com/questions/550038/is-it-valid-to-replace-http-with-in-a-script-src-http

TL;DR uses http/https depending on which the client is requesting the source page with

Comment (jakect) [inline index.html]: Yeah, I think I was running into issues due to using `file://` before I setup the node server. Should no longer be an issue.

**Comment by jakect** [comment]
+1 good cleanup and optimizations.
